src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/abstractclassname/Example2.java

```diff
@@ -1,19 +1,18 @@
 /*
 AbstractClassName
-format = (default)^Abstract.+$
 ignoreModifier = true
-ignoreName = (default)false
 
 
 */
 
 package com.puppycrawl.tools.checkstyle.checks.naming.abstractclassname;
 
-class Example2 {
+class Example2 { // xdoc section -- start
   abstract class AbstractFirst {}
```

Comment:
Please remove xdoc section, we are not ready

Reply:
Why not? It is less work to add them now while transferring examples to tests?

Reply:
Hmm, in this case make them to be on its own line

Reply:
Makes sense. On it.